src/Alarm.cpp

```diff
@@ -24,7 +24,6 @@ void Alarm::calibrate(){
     alertSuccessfulAction();
     _isCalibrated = true;
   }
-  Serial.println("Alarm is calibrating");
```

Comment:
You won't even need to rebase for this one. That pull request is for this branch so it will automagically fix itself once it is merged.